patches/server/0978-Add-a-check-if-the-file-exists-to-prevent-to-overrid.patch

```diff
@@ -0,0 +1,29 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheMeinerLP <p.glanz@madfix.me>
+Date: Tue, 13 Jun 2023 16:10:59 +0200
+Subject: [PATCH] Add a check if the file exists to prevent to override the
```

Comment:
Incorrect patch subject. This should describe what the patch does, not how.
`Add a check if the file exists to prevent to override the file each time` says absolutely nothing :sweat_smile:

Reply:
I will be change that. Thanks for notice that